recipes/spingo/meta.yaml

```diff
@@ -0,0 +1,25 @@
+about:
+  home: https://github.com/GuyAllard/SPINGO
+  license: ''
+  summary: 'SPecies level IdentificatioN of metaGenOmic amplicons'
+build:
+  number: 0
+  string: py{{CONDA_PY}}_boost{{CONDA_BOOST}}
+package:
+  name: spingo
+  version: 'v1.3'
+requirements:
+  build:
+      - gcc
+      - libgcc >=3.4.15
```

Comment:
can you follow our guideline here: https://bioconda.github.io/guidelines.html#c-c